src/NServiceBus.Core/Sagas/SimpleSaga.cs

```diff
@@ -0,0 +1,87 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Linq.Expressions;
+
+    /// <summary>
+    /// This class is used to define sagas containing data and handling a message.
+    /// To handle more message types, implement <see cref="IHandleMessages{T}" />
+    /// for the relevant types.
+    /// To signify that the receipt of a message should start this saga,
+    /// implement <see cref="IAmStartedByMessages{T}" /> for the relevant message type.
+    /// </summary>
+    /// <typeparam name="TSagaData">A type that implements <see cref="IContainSagaData" />.</typeparam>
+    public abstract class SimpleSaga<TSagaData> : Saga where TSagaData : IContainSagaData, new()
```

Comment:
Slightly OT: Can a simple saga have custom finders associated with it? should we consider restriction to start pushing users towards finders provided natively by the persisters?